ballerina/Ballerina.toml

```diff
@@ -5,6 +5,7 @@ version = "1.2.0"
 authors = ["Ballerina"]
 keywords = ["java", "arrays"]
 repository = "https://github.com/ballerina-platform/module-ballerina-jballerina.java.arrays"
+icon = "resources/icon.png"
```

Comment:
Let's move icon to ballerina root directory

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-jballerina.java.arrays/pull/365/commits/e2c276e071b58e44862451f16bfebfd50d05e5f2